fars/api/views.py

```diff
@@ -24,6 +27,65 @@ class BookingsList(viewsets.ViewSetMixin, generics.ListAPIView):
     filter_backends = (filters.DjangoFilterBackend,)
     filter_class = BookingFilter
 
+    def filter_on_bookable(self, bookings, request):
+        bookable = request.query_params.get('bookable')
+        if bookable:
+            return bookings.filter(bookable__id_str=bookable), True
+        return bookings, False
+
+    def serialize_one(self, booking):
+        # If the booking is None, the serializer will create a null object, which we don't want
+        return Response(self.get_serializer_class()(booking).data if booking else None)
```

Comment:
Then what do we want? What will `None` produce if not also a null object? If the object doesn't exist, I think it should produce either null or a 404 response.

Reply:
Null object was probably not the right term to use. What I meant is that `serializer(None)` will produce a booking with all fields `null` (`{ name: null, start: null, end: null, ...})`, which to me does not seem like a valid booking. Therefore I opted for a null/empty response rather than an invalid one.

Reply:
That sounds reasonable. If the endpoint is only expected to return a single response, a 404 response might be more RESTful.